packages/hardhat/hardhat.config.js

```diff
@@ -25,6 +25,9 @@ module.exports = {
   networks: {
     local: {
       url: 'http://localhost:8545',
+      accounts: {
+        mnemonic: 'test test test test test test test test test test test junk',
```

Comment:
do we need this?

Reply:
It's not required but useful. I always need to open the hardhat wallet and need to go the docs. It's an easier way to found it.